### DISCUSSION TITLE
Why is Code - OSS started with the `--disable-gpu` flag?

### DISCUSSION THREAD

**Discussion by liam-teale** [prasanthrangan/hyprdots#1324] [Q&A] [answered]
Title

**Comment by kRHYME7** [comment, marked as answer]
You can remove that flag if needed some devices do not render code on wayland without that flag.

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/82049541-5a6c-42e1-9fe8-6a3e09bb5515)
